### PR TITLE
fix: Advisor names (array shift)

### PR DESCRIPTION
**Pull request by EttyKitty**

### CODE REVIEW

File: scripts/scr_draw_armentarium/scr_draw_armentarium.gml [hunk 1]
```diff
@@ -436,7 +436,7 @@ function scr_draw_armentarium(){
         draw_text_transformed(xx + 336 + 16, yy + 66, string_hash_to_newline(header), 1, 1, 0);
         if (!in_forge){
             draw_set_font(fnt_40k_30b);
-            draw_text_transformed(xx + 336 + 16, yy + 100, string_hash_to_newline("Forge Master " + string(obj_ini.name[0, 2])), 0.6, 0.6, 0);
+            draw_text_transformed(xx + 336 + 16, yy + 100, string_hash_to_newline("Forge Master " + string(obj_ini.name[0, 1])), 0.6, 0.6, 0);
```
Comment (OH296) [inline scripts/scr_draw_armentarium/scr_draw_armentarium.gml:439]:
```suggestion
             var _forgemaster = fetch_unit([0,1]);
            draw_text_transformed(xx + 336 + 16, yy + 100, _forgemaster.name_role(), 0.6, 0.6, 0);
```

**Review by OH296 — APPROVED** [review]
was gonna make more suggestions but realised i was simply being a pedant. All good to me

**Comment by OH296** [comment]
only thing i will say is seems like it would be useful to have some quick functions like get_forge_master etc to simplify some of this maybe i'll pr into this if i find a moment

File: scripts/scr_draw_armentarium/scr_draw_armentarium.gml (hunk 1 above)
Comment (EttyKitty) [reply scripts/scr_draw_armentarium/scr_draw_armentarium.gml:439]: I'll not commit this for now, as refactoring only one line won't do much good, I think.

**Comment by EttyKitty** [comment]
> only thing i will say is seems like it would be useful to have some quick functions like get_forge_master etc to simplify some of this maybe i'll pr into this if i find a moment

If you don't mind, I'll merge this for now. The point of the PR is to fix a bug, all further refactors can be done as separate PRs.

**Comment by OH296** [comment]
> > only thing i will say is seems like it would be useful to have some quick functions like get_forge_master etc to simplify some of this maybe i'll pr into this if i find a moment
> 
> If you don't mind, I'll merge this for now. The point of the PR is to fix a bug, all further refactors can be done as separate PRs.

no problems, my error, i should have followed with a comment to state i was not going to get around to doing anything

**Pull request merged**